at_client/lib/src/service/notification_service.dart

```diff
@@ -156,10 +156,12 @@ class NotificationParams {
   }
 
   /// Returns [NotificationParams] to send a text message to another atSign.
-  static NotificationParams forText(String text, String whomToNotify) {
+  static NotificationParams forText(String text, String whomToNotify,
```

Comment:
can we replace whomToNotify with recipient ?

Reply:
This might effect the backward compatibility of the app.